functions/github/github.js

```diff
@@ -112,20 +113,26 @@ async function getReleaseReport(octokit, releaseData) {
 }
 
 /**
- * Fetches and returns metadata for each library in the release configuration.
- * The metadata includes the updated version, whether the library is opted in
- * to the release, and whether the library is lockstep. A library is lockstep
- * if it is part of the release but has no changes. A library is opted in if
- * it was not originally part of the release report, but has been manually opted
- * in by being added to the release configuration.
+ * Fetches and returns metadata for each library in the release according to
+ * the list of libraries in the release, and the changes that we know about.
+ * Metadata includes whether the library is part of a group release, whether
+ * the library is opted in to the release, and the updated version of the
+ * library.
  *
- * Fetches and returns metadata for each library in the release according
- * to the list of libraries in libraryNames. The metadata includes the updated
- * version, whether the library is opted in to the release, and whether the
- * library is a part of a group release. A library is a part of a group release
- * if it is part of the release but has no changes.
- * A library is opted in if it is not in the libraryChanges, but is in the
- * libraryNames.
+ * A library is part of a group release if it has zero changes, but is included
+ * in the release.
+ *
+ * A library is opted in to a release if it is not included in our changes at
+ * all, but was included in the release. Similarly, a library is opted out of
+ * a release if it is included in our changes, but it is not included in the
+ * list of releases.
+ *
+ * Note: These inferences for metadata are made with the assumption that
```

Comment:
Unfortunately, this assumption is invalid. We update a release more often than not during a release, and the update is almost always a cherry pick of some sort that adds to changelogs- whether it be teams forgetting to add changelog entries to opt in, forgetting a specific changelog entry, or changing the wording on something.